context_chat_backend/controller.py

```diff
@@ -309,6 +311,11 @@ def _(query: Query) -> LLMOutput:
 		# todo: migrate to Depends during db schema change
 		llm: LLM = llm_loader.load()
 
+		try:
+			query.query = json.loads(query.query).get("prompt", query.query)
+		except JSONDecodeError:
+			pass
+
```

Comment:
the context_chat app should be adjusted to pass in the correct params. Are you using the branch with task processing support?

Reply:
No, not yet.

Reply:
It is in main now.

Reply:
With the merged PR this is no longer necessary, nice